### PR TITLE
Fixes to PR Deployer

### PR DESCRIPTION
**Pull request by FaisalAbid**
This fixes

1. Properly OAuth Generated GitHub Token (added to Travis already)
2. Bug where two branches would be deployed on the same instance.

### CODE REVIEW

File: tool/prdeployer.dart [hunk 1]
```diff
@@ -41,14 +41,14 @@ main(List<String> arguments) async {
 
   if (projectToDeploy == null) {
     await projectsData.forEach((String key, Map<String, String> value) async {
-      if (projectToDeploy != null) {
-        // project already set
-        // todo: remove this foreach with a better iterator pattern
+      if(projectToDeploy != null){
```
Comment (sethladd) [inline tool/prdeployer.dart]: space after if

File: tool/prdeployer.dart [hunk 2]
```diff
@@ -41,14 +41,14 @@ main(List<String> arguments) async {
...
         return;
       }
-      projectToDeploy = value["name"];
-      await fbClient.patch("${firebaseRoot}/$key.json", {"branch": branchName});
-      await postLinkToGithub(projectToDeploy, currentPullRequest);
+      if(value["branch"] == null){
```
Comment (sethladd) [inline tool/prdeployer.dart]: space after if

Or, just run the dartfmt tool :)

Comment (FaisalAbid) [inline tool/prdeployer.dart]: Ah i wonder why webstorm didnt run the tool when I saved. Good catch! Fixing

**Comment by sethladd** [comment]
Yeah, WebStorm doesn't run dartfmt automatically. Maybe there's a way to trigger that?

**Comment by FaisalAbid** [comment]
Oh I know that, I have it set to cmd shift -f and I do it right before save. Its become a habit now to just hit it and save Lol, I guess it didnt work that time!

**Comment by sethladd** [comment]
lgtm

**Comment by sethladd** [comment]
Please merge in master, it has a fix so this PR will pass the linkchecker. Thanks!

**Comment by FaisalAbid** [comment]
Sorry! missed your comment in my Github noise!

**Comment by sethladd** [comment]
OK, here we go! lgtm :)

**Pull request merged**